website/site/content/pages/community.md

```diff
@@ -16,6 +16,8 @@ howtojoin: |
   **On the web:**
 
   1. https://bluejeans.com/278173690
+  
+  2. We also have a [Gitter](https://gitter.im) channel where you can get help and give a hand to others: https://gitter.im/netlify/NetlifyCMS
```

Comment:
Thanks @thomaskuntzz! I'm not sure that this is the right place to put this, though, since this section is specifically for how to join our bi-weekly planning meetings.

@erquhart @neutyp @ziburski Any suggestions on where this would work best?

Reply:
Agreed, somewhere separate from the Dev Meeting stuff, and more prominent.

Reply:
I think it would be good to have it on /docs/contributor-guide/ as the top item in the bullet list.

Reply:
Agreed, I approved your PR for that. I'd like it mentioned on both pages, can't have enough exposure for that.